kobo/settings/base.py

```diff
@@ -714,7 +714,7 @@ def __init__(self, *args, **kwargs):
         AZURE_ACCOUNT_KEY = env.str('AZURE_ACCOUNT_KEY')
         AZURE_CONTAINER = env.str('AZURE_CONTAINER')
         AZURE_URL_EXPIRATION_SECS = env.int('AZURE_URL_EXPIRATION_SECS', None)
-        AZURE_OVERWRITE_FILES = True
+        AZURE_OVERWRITE_FILES = env.bool('AZURE_OVERWRITE_FILES', False)
```

Comment:
What's a use case where an admin would choose to set to True - and thus break their own kobo instance? If there is no use case, then delete this line. The [default](https://django-storages.readthedocs.io/en/latest/backends/azure.html) is False.

Reply:
Gone!